SDVersion/SDiOSVersion/SDiOSVersion.m

```diff
@@ -89,6 +96,22 @@ + (NSDictionary*)deviceNamesByCode
                               @"iPad7,2"  : @(iPadPro12Dot9Inch2Gen),
                               @"iPad7,3"  : @(iPadPro10Dot5Inch),
                               @"iPad7,4"  : @(iPadPro10Dot5Inch),
+                              @"iPad7,5"  : @(iPad6),
+                              @"iPad7,6"  : @(iPad6),
+                              @"iPad7,11" : @(iPad7),
+                              @"iPad7,12" : @(iPad7),
+                              @"iPad8,1"  : @(iPadPro11Inch),
+                              @"iPad8,2"  : @(iPadPro11Inch),
+                              @"iPad8,3"  : @(iPadPro11Inch),
+                              @"iPad8,4"  : @(iPadPro11Inch),
+                              @"iPad8,5"  : @(iPadPro12Dot9Inch3Gen),
+                              @"iPad8,6"  : @(iPadPro12Dot9Inch3Gen),
+                              @"iPad8,7"  : @(iPadPro12Dot9Inch3Gen),
+                              @"iPad8,8"  : @(iPadPro12Dot9Inch3Gen),
+                              @"iPad11,1" : @(iPadMini5),
+                              @"iPad11,2" : @(iPadMini5),
+                              @"iPad11,4" : @(iPadAir3),
+                              @"iPad11,5" : @(iPadAir3),
```

Comment:
@"iPad11,3" : @(iPadAir3),   // wifi version
@"iPad11,4" : @(iPadAir3),

Aren't they the machine ids of iPad Air 3rd Gen?